presto-native-execution/presto_cpp/main/PrestoTask.cpp

```diff
@@ -417,7 +417,7 @@ protocol::TaskInfo PrestoTask::updateInfoLocked() {
   prestoTaskStats.endTime = util::toISOTimestamp(taskStats.executionEndTimeMs);
 
   const uint64_t currentTimeMs = velox::getCurrentTimeMs();
-  const uint64_t sinceLastPeriodMs = currentTimeMs - lastTaskStatsUpdateMs;
+  const double sinceLastPeriodMs = currentTimeMs - lastTaskStatsUpdateMs;
```

Comment:
Would you explain what was happening before this change?

Wondering if we can add some tests to prevent this from breaking in the future.

Reply:
Prestissimo started to see a special value on cumulative memory usage report which was a double in Java but uint64_t in cpp. I suspect this is an overflow.

Reply:
How can we overflow 'ms'?
This is not possible, I believe.
If we see large numbers - it is likely due to a bug.

Reply:
It is time interval * average memory usage

Reply:
Ok, this is a regression at Feb22 when we started to see bad stats.